core/templates/pages/blog-admin-page/blog-admin-page.module.ts

```diff
@@ -16,8 +16,9 @@
  * @fileoverview Module for the blog-admin page.
  */
 
-import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
-import { APP_INITIALIZER, DoBootstrap, NgModule } from '@angular/core';
+// eslint-disable-next-line oppia/disallow-httpclient
```

Comment:
Ditto, please add an explanation so that people don't think this is a common pattern.

Or should we update our eslint config to not apply this check to module.ts files? That might be better.

Reply:
Providing `HttpClient` as a dependency in module files is a common practice. We should go with the second approach. /cc @srijanreddy98

Reply:
Ok, can you do that please and drop the in-file exceptions?

Reply:
We can still sort of avoid it with my comment down below (https://github.com/oppia/oppia/pull/15783#pullrequestreview-1048650677). But that only works because it is common across all files. If this was a module specific thing, we would have to update our eslint config.

Reply:
Done as per @srijanreddy98 's suggestion.